application/models/Participant.php

```diff
@@ -24,7 +24,7 @@
  * @property string $lastname
  * @property string $email
  * @property string $language
- * @property integer $blacklisted
+ * @property string $blacklisted
```

Comment:
integer or string but use it at boolean  : 

- 0 == null == "" : not blacklisted. 
- 1 == "Y": blacklisted

Reply:
See 3.X
![Capture d’écran du 2022-02-08 16-20-19](https://user-images.githubusercontent.com/1439428/153017711-c062a405-39d2-4c9b-b166-a05beba09d33.png)

False is save as `0` , and true save as `Y` . Then  we can cast it as bookean 

`boolval("0") === boolval(0) === false` , boolval("Y") === true

No issue 
https://www.php.net/manual/fr/function.boolval.php

Reply:
😵